Core/clim-core/presentation-defs.lisp

```diff
@@ -1382,8 +1387,11 @@ protocol retrieving gestures from a provided string."))
 
 (define-presentation-method accept ((type real) stream (view textual-view)
                                     &key)
-  (let ((*read-base* base))
-    (accept-using-read stream type)))
+  (let ((*read-base* base)
+	(read-result (accept-using-read stream type)))
+    (if (and (null read-result) default)
+	(values default type)
```

Comment:
no worries. You may add new commit to the same master branch, we use --no-ff option, so it won't clobber main branch anyways.

As of indenter, I use emacs+slime default indentation with McCLIM loaded (and I have `(setq-default indent-tabs-mode nil)` in my init.el)